deploy.Jenkinsfile

```diff
@@ -4,8 +4,8 @@ def repo_owner = "pace-neutrons"
 def release_id_description = (
   "The IDs of the jobs that contain the target release artifacts.\n" +
   "This parameter should have the form:\n\n" +
-  "  <JOB_NAME>, <BUILD_NUMBER>;\n" +
-  "  <JOB_NAME>, <BUILD_NUMBER>;\n" +
+  "  <PIPELINE_NAME>, <BUILD_NUMBER>;\n" +
+  "  <PIPELINE_NAME>, <BUILD_NUMBER>;\n" +
```

Comment:
Something I missed here before, but wrapping `PIPELINE_NAME`/`BUILD_NUMBER` in `< >` actually acts to make them a comment, so they're not rendered by Jenkins. It may be best to just remove the `< >`s.

![image](https://user-images.githubusercontent.com/46960113/105861448-a462fe80-5fe6-11eb-804c-bee2ffa2d07a.png)